tests/test_strategies.py

```diff
@@ -401,18 +401,20 @@ def test_distributed_C3_prioritization(self):
         input = TEST_REPO_PATH / 'test_data/input_test_strategies/scenario_C3.json'
         s = scenario.Scenario(load_json(input), input.parent)
         s.run('distributed', {"testing": True})
-        # GC power must always be max (use battery)
+        # GC power must have reached maximum
+        assert pytest.approx(s.testing["max_total_load"]) == 5
+        # GC power must always be max or zero
         for load in s.testing["timeseries"]["total_load"]:
-            assert pytest.approx(load) == pytest.approx(s.testing["max_total_load"]) == 5
+            assert pytest.approx(load) == 0 or pytest.approx(load) == 5
         cs = s.testing["timeseries"]["sum_cs"]
         cs_1 = [x for x in cs if x[0] != 0]
         cs_2 = [x for x in cs if x[1] != 0]
         # only one cs at a time
         assert [x[1] == 0 for x in cs_1]
         assert [x[0] == 0 for x in cs_2]
         # assert that vehicles are charged balanced
-        assert len(set([round(x[0], 2) for x in cs_1])) == 1
-        assert len(set([round(x[1], 2) for x in cs_2])) == 1
+        # assert len(set([round(x[0], 2) for x in cs_1])) == 1
+        # assert len(set([round(x[1], 2) for x in cs_2])) == 1
```

Comment:
Why are these two lines a comment and not deleted?

Reply:
uncommented first, removed second assert (GC power not sufficient for both to charge balanced in estimated time)